content/dstu2/observation.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Observation | FHIR DSTU 2 API
+---
+
+# Observation
+
+* TOC
+{:toc}
+
+## Search
+
+Search for labs, vitals, and alcohol/tobacco use Observations that meet supplied query parameters:
+
+    GET /Observation?:parameters
+
+### Parameters
+
+Name | Type | Description
+-----|------|--------------
+`subject`|`reference`| The subject that the observation is about.
+`patient`|`reference`| The subject that the observation is about (if patient).
```

Comment:
Nit: list params in the same order as the spec - http://hl7.org/fhir/2015May/observation.html#4.20.6